app/src/Components/Project/ProjectFullDetails.tsx

```diff
@@ -90,10 +92,7 @@ const ProjectFullDetails: FC<ProjectFullDetailsProps> = ({ project }) => {
             icon={<Feather name='file' size={24} />} 
             details={ "A3 - Project Scope"}
           />
-          <ProjectAttachments
-            icon={<Feather name='video' size={24} />} 
-            details={ "Charity Video" }
-          />
+          <Video details={project.video_file} />
```

Comment:
The video wouldn't show in the app for me until I changed `details` here to `url` -- as this parameter's named `url` in `VideoProps` in the file below